src/landing-page/landing-page.js

```diff
@@ -0,0 +1,252 @@
+						{ __(
+							'Start from scratch! Create a blank theme to get started with your own design ideas.',
+							'create-block-theme'
+						) }
+					</p>
+					<Button
+						variant="link"
+						onClick={ () => setCreateModalType( 'clone' ) }
+					>
+						{ sprintf(
+							// translators: %s: theme name.
+							__(
+								'Create a Clone of "%s"',
+								'create-block-theme'
+							),
+							themeStyleData?.name
+						) }
+					</Button>
+					<p>
+						{ __(
+							'Use the currently activated theme as a starting point.',
+							'create-block-theme'
+						) }
+					</p>
+					<Button
+						variant="link"
+						onClick={ () => setCreateModalType( 'child' ) }
+					>
+						{ sprintf(
+							// translators: %s: theme name.
+							__(
+								'Create a Child of "%s"',
+								'create-block-theme'
+							),
+							themeStyleData?.name
+						) }
+					</Button>
+					<p>
+						{ __(
+							'Make a theme that uses the currently activated theme as a parent.',
+							'create-block-theme'
+						) }
+					</p>
+				</VStack>
+				<VStack className="cbt-lp-right-column">
+					<h4>{ __( 'About the Plugin', 'create-block-theme' ) }</h4>
+					<p>
+						{ __(
+							"Create Block Theme is a tool to help you make Block Themes using the WordPress Editor. It adds tools to the Editor to help you create and manage your theme. Themes created with Create Block Theme are built using the WordPress Editor and are compatible with the Full Site Editing features in WordPress. Themes created with Create Block Theme don't require Create Block Theme to be installed on the site where the theme is used.",
+							'create-block-theme'
+						) }
+					</p>
+					<h4>
+						{ __( 'Do you need some help?', 'create-block-theme' ) }
+					</h4>
+					<p>
+						{ createInterpolateElement(
+							__(
+								'Have a question? Ask for some help in the <a>forums</a>.',
+								'create-block-theme'
+							),
+							{
+								a: (
+									// eslint-disable-next-line jsx-a11y/anchor-has-content
+									<a href="https://wordpress.org/support/plugin/create-block-theme/" />
+								),
+							}
+						) }
+					</p>
+					<p>
+						{ createInterpolateElement(
+							__(
+								'Found a bug? Report it on <a>GitHub</a>.',
+								'create-block-theme'
+							),
+							{
+								a: (
+									// eslint-disable-next-line jsx-a11y/anchor-has-content
+									<a href="https://github.com/WordPress/create-block-theme/issues/new" />
+								),
+							}
+						) }
+					</p>
+					<p>
+						{ createInterpolateElement(
+							__(
+								'Want to contribute? Check out the <a>project on GitHub</a>.',
+								'create-block-theme'
+							),
+							{
+								a: (
+									// eslint-disable-next-line jsx-a11y/anchor-has-content
+									<a href="https://github.com/WordPress/create-block-theme" />
+								),
+							}
+						) }
+					</p>
+					<div className="cbt-lp-faq">
+						<h4>{ __( "FAQ's", 'create-block-theme' ) }</h4>
```

Comment:
```suggestion
						<h4>{ __( "FAQ", 'create-block-theme' ) }</h4>
```

FAQ is already plural.